tla/README.md

```diff
@@ -5,3 +5,9 @@ This directory contains some formal specifications of CCF. For more information,
 You can also interact with these specifications using codespaces:
 
 [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=180112558&machine=xLargePremiumLinux&devcontainer_path=.devcontainer%2Ftlaplus%2Fdevcontainer.json&location=WestEurope)
+
+## Trace validation
```

Comment:
Might be worth putting this in the docs instead

Reply:
Ok, I'm moving everything then.